sdk/confidentialledger/azure-security-confidentialledger/src/test/java/com/azure/security/confidentialledger/generated/GetLedgerEntriesTests.java

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+
+package com.azure.security.confidentialledger.generated;
+
+import com.azure.core.http.rest.PagedIterable;
+import com.azure.core.http.rest.RequestOptions;
+import com.azure.core.http.rest.Response;
+import com.azure.core.util.BinaryData;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.IOException;
+import java.util.Iterator;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public final class GetLedgerEntriesTests extends ConfidentialLedgerClientTestBase {
+    @Test
+    public void testGetLedgerEntriesTests() {
```

Comment:
these tests seems not recorded https://github.com/Azure/azure-sdk-for-java/pull/29358/checks?check_run_id=7202429473